src/components/LocationSearch/styles.js

```diff
@@ -0,0 +1,90 @@
+import styled, { css } from "styled-components";
+
+export const LocationListWrapper = styled.div`
+  max-height: 90%;
+  overflow: auto;
+  padding-right: 1rem;
+  margin-top: var(--padding-size-medium);
+`;
+
+export const LocationSearchWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  padding-top: var(--font-size-medium);
+  top: 0;
+  left: 0;
+  z-index: 100;
+  padding: var(--padding-size-medium);
+  background-color: var(--color-primary);
+  transform: translateX(-100%);
+  transition: all ease-in-out 0.4s;
+
+  ${(props) =>
+    props.$isLocationSearchOpen &&
+    `
+      transform: translateX(0);
+    `}
+`;
+
+export const CloseTrigger = styled.button`
+  background: transparent;
+  width: 1.084375rem;
+  position: absolute;
+  right: 2rem;
+  top: 2rem;
+  height: 2rem;
+  width: 2rem;
+  color: var(--color-action-primary);
+`;
+
+export const LocationSearchForm = styled.form`
+  width: 100%;
+  display: flex;
+  gap: 0.5rem;
+  margin-top: 2.5rem;
+`;
+
+const commonHeight = css`
+  height: 3rem;
+`;
```

Comment:
That's so small that's better just write the line directly to each than writing this.
We usually reuse when it's something bigger (at least 2-3 rules)